www/source/demo/summary.html.slim

```diff
@@ -7,56 +7,52 @@ classes: demo_summary
   .row
     .columns.large-12.header
       img.habicat-congrats src="/images/demo/summary-congrats.svg"
-      h1.mb-2 Super powers activated!
-      h4.mb-2 Now that you've built and deployed your application with Habitat Builder, you can:
-  .row.mb-1
-    .columns.large-4.demo-summary--step-img: img src="/images/demo/summary-left.svg"
-    .columns.large-4.demo-summary--step-img: img src="/images/demo/summary-middle.svg"
-    .columns.large-4.demo-summary--step-img: img src="/images/demo/summary-right.svg"
-  .row
-    .columns.large-4
-      .demo-summary--item
-        p Consume automatically updated builds every time Habitat Builder detects a new GitHub commit or underlying package dependency change.
-    .columns.large-4
-      .demo-summary--item
-        p Export the same immutable Habitat package to a variety of formats (Docker, ACI, bare metal, Cloud Foundry, Kubernetes, and more) to power your different workloads and environments.
-    .columns.large-4
-      .demo-summary--item
-        p Use the Habitat Supervisor to keep your application healthy and happy over its lifetime via configuration updates, automated update strategies, health checks, and more.
-    .row.mb-2
-      .columns.large-4
-        .demo--callout-content
-          p: a.link-external href="/tutorials/get-started/demo/" target="_blank" Try the Habitat Runtime Demo
-          p.text-gray Get hands on with the Supervisor managing application services!
-      .columns.large-4
-        .demo--callout-content
-          p: a.link-external href="/tutorials/download/" target="_blank" Download and Install the CLI tool
-          p.text-gray Build and run artifacts from plan files using the Habitat Studio on your CLI.
-      .columns.large-4
-        .demo--callout-content
-          p: a.link-external href="/tutorials/build-your-own/" target="_blank" Build your Application from the Ground Up
-          p.text-gray Build, deploy, and manage a full-stack web application using Scaffolding!
+      h1.mb-2 Which Habitat capabilities<br>would you like to unlock next?
+      h3.mb-2 Learn how you can build, deploy, and manage your applications with Habitat.
+      p
+        a href="/demo" Return to demos
```

Comment:
I would make this bigger, possibly a UI element similar to the ones below it instead, I couldn't find it and was confused where the other demo modules have gone. Or we could just take you straight back to the demos landing page instead of this next steps page.